### DISCUSSION TITLE
Id of an action

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#9293] [Help]
Hello everyone.
After filling in the form and saving the data in the database (action2), I want to create a new row in a new table and give the row Id of (action2) to the relational column between the two tables.
To do this i used this expression : **{{ Action 2.Saved row._id }}**

![action](https://user-images.githubusercontent.com/106697208/211451564-602dca40-f413-4e7b-b564-6ed4eb86c4b7.PNG)

but there is an error, when I fill in the form and I click on save

![error](https://user-images.githubusercontent.com/106697208/211451853-b98c4c35-d2f2-456d-b564-140de761bae8.PNG)

@melohagan help please.

**Comment by melohagan** [comment]
Hey @rafikApp 

Could you share a screenshot of Action 2 please?

**Reply by rafikApp** [reply]
Hey  @melohagan 
![Capture](https://user-images.githubusercontent.com/106697208/211682339-1390859b-97dd-47d2-a125-0044b5dbe350.PNG)

**Reply by melohagan** [reply]
I think because you are using the Form datasource in both Save Row actions that it is possibly saving duplicate/conflicting relationship data.

When you click `Add Column` that is an override. The other fields in the Form schema will also be saved too.

**Reply by rafikApp** [reply]
No it does not save duplicate relation data, because if I arrive at the save row 4 step then I am sure that the relational column between the two tables (action 2 and 4) is empty. So there will be no problem if I give it the id of action 2.
